Libraries/LibGUI/GTextEditor.cpp

```diff
@@ -29,6 +29,15 @@ GTextEditor::GTextEditor(Type type, GWidget* parent)
     vertical_scrollbar().set_step(line_height());
     m_cursor = { 0, 0 };
     create_actions();
+
+    // Note: This constructor gets called more than one during initalization. Bug?
```

Comment:
If you are testing with the `TextEditor` application, there is more than one `GTextEditor`.
The "find" input box is a `GTextBox` which inherits from `GTextEditor`. So this is not a bug :)